test/specs/lightbox.js

```diff
@@ -1,11 +1,66 @@
-/* eslint-env jasmine */
-
+import $ from 'jquery';
 import lightbox from '../../lightbox';
 
 describe('lightbox', function() {
-  describe('init', function() {
-    it('should be a function', function() {
-      expect(lightbox.init).toEqual(jasmine.any(Function));
+  const FADE_TIME = 200;
+  const BLOCKING_CLASS = '.js-blocking';
+  const LIGHTBOX_CLASS = '.js-lightbox';
+  const NEXT_CLASS = '.js-next';
+  const CLOSE_CLASS = '.js-close';
```

Comment:
Any reason for using these as "real" constants with upper/snake case?

We should probably discuss standardizing the var formats in tests, there's been a few variations in our code base including below:

`this._hello = world`

```js
describe( '' , () => { 
   let hello;

   it('world', () => {
    hello = world;
   }
});
```